style.css

```diff
@@ -558,6 +659,110 @@
     font-family: "Inter", sans-serif;
   }
 
+  .name-render {
+    font-family: 'Inter', sans-serif;
+    font-size: 35px;
+    text-align: left;
+  }
+
+  .pop-up-modal {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background-color: white;
+    padding: 0 1.5rem;
+    padding-bottom: 1.5rem;
+    border-radius: 0.5rem;
+  }
```

Comment:
- [ ] Kindly add some padding-top to your modal, because when I open it I can't see the close button and the top of it.

![image](https://user-images.githubusercontent.com/35535130/156467710-3c1ecedd-6486-4285-8247-b5c5ab6e28cd.png)

- [ ] And the same with the mobile version I can't see the close button.


![image](https://user-images.githubusercontent.com/35535130/156468145-fbe7bded-f5c9-4218-a869-1b25819fc4bd.png)

Reply:
Thank you so much for your requirements I didn't notice that but now its done :smile:

Reply:
- [ ] Well done on doing the requested changes from the previous reviewer but please remove that black line on the left of the page and now you have a big white space before the X button. 

Here you have a screenshot of the modal on the mobile screen:

![image](https://user-images.githubusercontent.com/74506933/156474596-ab1d9364-faaa-45d2-9cc7-5a1644b3c219.png)

Reply:
Thanks for the advice, now it's done.


![Screenshot from 2022-03-02 20-53-45](https://user-images.githubusercontent.com/96317674/156481375-af4e50ef-7433-46ed-88b5-8e06fe0f90b2.png)
the white space is not there anymore


![Screenshot from 2022-03-02 20-54-12](https://user-images.githubusercontent.com/96317674/156481494-4824f4fe-8d3f-401d-81eb-174180430507.png)